src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const totalCost = dailyPrice * days;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return totalCost - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return totalCost - shortTermDiscount;
```

Comment:
There is a logical issue in the discount application. If the number of days is greater than or equal to `longTerm`, the function will apply the long-term discount and return immediately, without checking the short-term discount condition. This means that if both conditions are met, only the long-term discount will be applied. You may want to reconsider the logic to ensure the correct discount is applied based on the number of days.